### PR TITLE
Applies crystal tool format to master branch

### PR DESCRIPTION
**Pull request by eliasjpr**
## Applies crystal tool format to master branch

Issue: https://app.clubhouse.io/amber/story/45/apply-tool-format-to-master

Some files needed to be formatted by the crystal tool format

* Applied crystal tool formats and formatted a couple of files.

* Cleans up the formatting in static_controller, render, router dsl, and
route class.

### CODE REVIEW

File: src/amber/controller/render.cr [hunk 1]
```diff
@@ -1,21 +1,21 @@
 module Amber::Controller
   module Render
-      macro render_both(filename, layout)
+    macro render_both(filename, layout)
         content = render_template("{{filename.id}}")
```
Comment (bew) [inline src/amber/controller/render.cr]: Macros needs manual indentation

File: src/amber/controller/render.cr [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
         render_template("{{layout.id}}")
       end
 
-      # helper to render a template.  The view name is relative to `src/views` directory.
-      macro render_template(filename, *args)
+    # helper to render a template.  The view name is relative to `src/views` directory.
+    macro render_template(filename, *args)
         {% if filename.id.split("/").size > 2 %}
```
Comment (bew) [inline src/amber/controller/render.cr]: Macros needs manual indentation

File: src/amber/controller/render.cr [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
           Kilt.render("{{filename.id}}", {{*args}})
         {% else %}
           Kilt.render("src/views/{{filename.id}}", {{*args}})
         {% end %}
       end
 
-      macro render(filename, layout = "layouts/application.slang", path = "src/views", folder = __FILE__)
+    macro render(filename, layout = "layouts/application.slang", path = "src/views", folder = __FILE__)
         render_both "#{{{path}}}/#{{{folder.split("/").last.gsub(/\_controller\.cr|\.cr/, "")}}}/#{{{filename}}}", "#{{{path}}}/#{{{layout}}}"
```
Comment (bew) [inline src/amber/controller/render.cr]: Macros needs manual indentation

File: src/amber/controller/render.cr (hunk 1 above)
Comment (eliasjpr) [inline src/amber/controller/render.cr]: 👍

File: src/amber/controller/render.cr (hunk 2 above)
Comment (eliasjpr) [inline src/amber/controller/render.cr]: 👍

File: src/amber/controller/render.cr (hunk 3 above)
Comment (eliasjpr) [inline src/amber/controller/render.cr]: 👍

**Comment by eliasjpr** [comment]
Please review

**Pull request merged**